plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -61,6 +63,10 @@
  * @author Thomas Mäder
  */
 public class JavaLanguageServerExtensionService {
+
+  private static final String FILE_STRUCTURE = "java/file-structure";
```

Comment:
Using a constant is not productive for something that is only used once. A constant would be useful if we can share it between the back end and front end. In this way, we're just removing the value from the place it's used.